droid-hal-device.inc

```diff
@@ -535,6 +546,20 @@ android_version_major=$(awk '/PLATFORM_VERSION([A-Z0-9.]*|_LAST_STABLE) := ([0-9
                           < %{android_root}/build/core/version_defaults.mk \
                           | awk -F'.' '{print $1}')
 
+# Find the kernel build directory.
+# AOSP seems to use .../obj/kernel/.config not obj/KERNEL_OBJ/.config like CM.
+# Sony AOSP 11 uses a separate storage area.
```

Comment:
There isn't really a standard for these as you are not supposed to be build the kernel inside the Android build system.

Also the comment about CM is out of date as there's no CM anymore